sdk/storage/azure-storage-blob-changefeed/pom.xml

```diff
@@ -67,7 +67,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob</artifactId>
-      <version>12.25.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
+      <version>12.25.0</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
```

Comment:
We are **not** GAing ChangeFeed, please undo this.

Reply:
These changes are only updating the dependent package verisons, azure-storage-blob and azure-storage-common. These version updates are needed or Changefeed will be depending on previous versions of blob and common